uvc-src/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "uvc-src"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
I've rather this be a `0.3` instead, to prevent this to accidentally flow down to existing users. 
Same goes for `-sys` and the main crate

Reply:
done